salt/cloud/clouds/virtualbox.py

```diff
@@ -278,7 +277,7 @@ def list_nodes_select(call=None):
     """
     Return a list of the VMs that are on the provider, with select fields
     """
-    return cloud.list_nodes_select(
+    return __Utils__['cloud.list_nodes_select'](
```

Comment:
This should be `__utils__` instead of `U`.

Reply:
good catch!